plugins/packages/airtable/lib/manifest.json

```diff
@@ -15,13 +15,20 @@
       "api_key": {
         "type": "string",
         "encrypted": true
+      },
+      "personal_access_token": {
+        "type": "string",
+        "encrypted": true
       }
     },
     "customTesting": true
   },
   "defaults": {
     "api_key": {
       "value": ""
+    },
+    "personal_access_token": {
+      "value": ""
     }
   },
   "properties": {
```

Comment:
we can remove `api_key` such that it will still work until jan for those who don't update
but at connection options with this change we only use PAT